ivy/container/elementwise.py

```diff
@@ -4469,6 +4484,21 @@ def logaddexp(
             a container containing the element-wise results. The returned container
             must have a real-valued floating-point data type determined
             by :ref:`type-promotion`.
+
+        Examples
+        --------
+        Using :code:`ivy.Container` input:
+        
+        >>> x = ivy.Container(a=ivy.array([4, 5, 6]),\
```

Comment:
Please update this example to have all values as floats.

Reply:
changed all values to floats